modules/products/rider/src/main/kotlin/com/metalbear/mirrord/products/rider/RiderLaunchSettingsExtension.kt

```diff
@@ -0,0 +1,39 @@
+package com.metalbear.mirrord.products.rider
+
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.openapi.project.Project
+import com.jetbrains.rd.util.lifetime.Lifetime
+import com.jetbrains.rider.run.configurations.RiderConfigurationLaunchSettingsExtension
+import com.jetbrains.rider.run.configurations.RuntimeHotReloadRunConfigurationInfo
+import com.jetbrains.rider.run.configurations.launchSettings.LaunchSettingsJson
+import com.metalbear.mirrord.CONFIG_ENV_NAME
+import java.util.concurrent.ConcurrentHashMap
+
+class RiderLaunchSettingsExtension : RiderConfigurationLaunchSettingsExtension {
```

Comment:
Could you add some docs for these methods, saying that we need them to solve issue N, get the MIRRORD_CONFIG env and stuff?

Reply:
yes will add, thank you!